File: vlingo-eventjournal/src/test/java/io/vlingo/eventjournal/ActorTest.java
package io.vlingo.eventjournal;

import io.vlingo.actors.World;
import org.junit.After;
import org.junit.Before;

public abstract class ActorTest {
    private World world;

    @Before
    public void setUpWorld() throws Exception {
        world = World.startWithDefaults("test");
    }

    @After
    public void tearDownWorld() throws Exception {
        world.terminate();
    }

    protected final World world() {
        return world;
    }
}

File: xoom-iot-android/app/src/test/java/com/example/vlingohike/ActorTest.kt
package com.example.vlingohike

import io.vlingo.actors.testkit.TestWorld
import org.junit.After
import org.junit.Before

abstract class ActorTest {
    private lateinit var testWorld: TestWorld

    @Before
    fun setUpWorld() {
        testWorld = TestWorld.startWithDefaults(javaClass.simpleName)
    }

    @After
    fun tearDownWorld() {
        testWorld.terminate()
    }

    protected fun world() = testWorld.world()
}